WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayCompleteViewModel.swift

```diff
@@ -14,6 +14,8 @@ final class SetUpTapToPayCompleteViewModel: PaymentSettingsFlowPresentedViewMode
     private let connectionAnalyticsTracker: CardReaderConnectionAnalyticsTracker
     private let stores: StoresManager
 
+    private let analytics: Analytics = ServiceLocator.analytics
```

Comment:
Something for a future PR: I'm not sure if is worth at the moment, as only `doneTapped` is public, but we could inject analytics into the constructor to make track events testable.